codec-http/src/main/java/io/netty/handler/codec/spdy/SpdyFrameCodec.java

```diff
@@ -122,7 +122,83 @@ protected SpdyFrameCodec(SpdyVersion version, int maxChunkSize,
             SpdyHeaderBlockDecoder spdyHeaderBlockDecoder, SpdyHeaderBlockEncoder spdyHeaderBlockEncoder,
             boolean validateHeaders, boolean supportsUnknownFrames) {
         this.supportsUnknownFrames = supportsUnknownFrames;
-        spdyFrameDecoder = createDecoder(version, this, maxChunkSize);
+        spdyFrameDecoder = createDecoder(version, supportsUnknownFrames ? new SpdyFrameDecoderExtendedDelegate() {
```

Comment:
The problem with this one is one more allocation.

Reply:
its one per codec... So I don't worry about it